CHANGELOG_iOS.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.76.77](https://github.com/brave/brave-browser/releases/tag/v1.76.77)
+
+ - Added Brave Translate for page translations. ([#40782](https://github.com/brave/brave-browser/issues/40782))
+ - Added generalization for the model names in Leo chat. ([#44195](https://github.com/brave/brave-browser/issues/44195))
+ - Added warning text in the setting description when user enables “Block all cookies”. ([#44295](https://github.com/brave/brave-browser/issues/42295))
+ - Added the ability to select custom app icons for Brave. ([#43141](https://github.com/brave/brave-browser/issues/43141))
+ - Disabled TLS 1.0 and TLS 1.1 due to versions being deprecated. ([#43819](https://github.com/brave/brave-browser/issues/43819))
+ - Fixed crash when enabling "Open Tabs" syncing. ([#44380](https://github.com/brave/brave-browser/issues/44380))
+ - Fixed parent tab permission pop-ups being shown on child tabs as reported on HackerOne by b4dc4t. ([#44061](https://github.com/brave/brave-browser/issues/44061))
+ - Fixed URL to be displayed properly when scanning QR codes as reported on HackerOne by roland_hack. ([#42559](https://github.com/brave/brave-browser/issues/42559))
```

Comment:
Lets change the above so it highlights the issue being fixed rather than mentioning it should be "displayed" correctly. Example:

```
 - Fixed URL being incorrectly displayed when scanning QR codes as reported on HackerOne by roland_hack. ([#42559](https://github.com/brave/brave-browser/issues/42559))
 ```